### ISSUE TITLE
doesnt work

### ISSUE THREAD

**Issue by Pikotaro** [Maddosaurus/b747400-sc#1]

**Comment by Pikotaro** [comment]
2019-02-06 03:16:01.156 INFO  Configuration file loaded
2019-02-06 03:16:01.156 WARN  Dataref type 7(int+float+double) will be used as float: laminar/B747/fuel/temperature
2019-02-06 03:16:01.156 INFO  Found: 0 CONTINUED integer DataRef's
2019-02-06 03:16:01.156 INFO  Found: 0 CONTINUED integer array DataRef's
2019-02-06 03:16:01.156 INFO  Found: 2 CONTINUED float DataRef's
2019-02-06 03:16:01.156 INFO  Found: 14 CONTINUED float array DataRef's
2019-02-06 03:16:01.156 INFO  Found: 0 CONTINUED string DataRef's
2019-02-06 03:16:01.156 WARN  Can't find DataRef: laminar/B747/engines/thrust_rev_lever_01
2019-02-06 03:16:01.156 WARN  Can't find DataRef: laminar/B747/engines/thrust_rev_lever_02
2019-02-06 03:16:01.156 WARN  Can't find DataRef: laminar/B747/engines/thrust_rev_lever_03
2019-02-06 03:16:01.156 WARN  Can't find DataRef: laminar/B747/engines/thrust_rev_lever_04
2019-02-06 03:16:01.156 WARN  Can't find DataRef: laminar/B747/engines/thrust_rev_lever_all
2019-02-06 03:16:01.156 WARN  Dataref type 7(int+float+double) will be used as float: laminar/B747/flt_ctrls/speedbrake_lever
2019-02-06 03:16:01.156 INFO  Found: 0 SEND_BACK integer DataRef's
2019-02-06 03:16:01.156 INFO  Found: 0 SEND_BACK integer array DataRef's
2019-02-06 03:16:01.156 INFO  Found: 6 SEND_BACK float DataRef's
2019-02-06 03:16:01.157 INFO  Found: 8 SEND_BACK float array DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 SEND_BACK string DataRef's
2019-02-06 03:16:01.157 INFO  Found: 3 SLOW integer DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 SLOW integer array DataRef's
2019-02-06 03:16:01.157 INFO  Found: 38 SLOW float DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 SLOW float array DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 SLOW string DataRef's
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/flight_controls/landing_gear_down
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/flight_controls/landing_gear_emer_off
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/flight_controls/landing_gear_emer_on
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/flight_controls/landing_gear_toggle
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/flight_controls/landing_gear_up
2019-02-06 03:16:01.157 WARN  Dataref type 7(int+float+double) will be used as float: laminar/B747/actuator/gear_handle
2019-02-06 03:16:01.157 WARN  Can't find DataRef: laminar/B747/gear_handle/position
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/FMS1/key_0
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/FMS2/key_0
2019-02-06 03:16:01.157 WARN  Can't find DataRef: laminar/B747/flt_ctrls/flap_lever
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/radios/nav1_standy_flip
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/radios/nav2_standy_flip
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/radios/adf1_standy_flip
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/radios/adf2_standy_flip
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/ground_ops/pushback_left
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/ground_ops/pushback_right
2019-02-06 03:16:01.157 WARN  Can't find DataRef: sim/ground_ops/pushback_straight
2019-02-06 03:16:01.157 INFO  Found: 21 TRIGGER integer DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 TRIGGER integer array DataRef's
2019-02-06 03:16:01.157 INFO  Found: 5 TRIGGER float DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 TRIGGER float array DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 TRIGGER string DataRef's
2019-02-06 03:16:01.157 INFO  Found: 0 CLICK DataRef's
2019-02-06 03:16:01.159 WARN  Can't find command - laminar/B747/fuel/fuel_jett/el_dial_dn
2019-02-06 03:16:01.161 WARN  Can't find command - laminar/B747/gear_handle/position
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/ias_mach/sel_dial_dn
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/ias_mach/sel_dial_up
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/ias_mach/sel_switch
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/button_switch/lnav_mode
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/button_switch/vnav_mode
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/heading/sel_dial_dn
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/heading/sel_dial_up
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/button_switch/hdg_sel_mode
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/button_switch/hdg_hold_mode
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/altitude/sel_dial_dn
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/altitude/sel_dial_up
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/button_switch/app_mode
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/switch/disengage_bar
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/bank_limit/sel_dial_dn
2019-02-06 03:16:01.162 WARN  Can't find command - laminar/B747/autopilot/bank_limit/sel_dial_up
2019-02-06 03:16:01.164 INFO  Found: 461 COMMANDS
2019-02-06 03:16:01.164 INFO  Found: 0 SHARED dataref's
2019-02-06 03:16:01.164 INFO  Found autopilot datarefs
2019-02-06 03:16:01.164 INFO  Autopilot sequenser is active
2019-02-06 03:16:01.164 INFO  Found: 2 OVERRIDE DataRef's

**Comment by Maddosaurus** [comment]
Yeah - it looks like it broke in one of the more recent updates, sorry.  
Thanks for providing the log, though!  
I'll have a look at the problems. In the meantime it would be wonderful if you could provide me with the versions of Smart Copilot and X-Plane you're using!
